app/apis/users.js

```diff
@@ -16,7 +16,7 @@ function addUser(email, password, name) {
 }
 */
 async function addAdmin(email, password, name) {
-    return await insertUser(email, password, name, 'admin');
```

Comment:
@brandngo just note that this is more efficient.

async functions will automatically resolve all promises given to it and then wrap it in another promise. :)